meta-intel-edison-bsp/recipes-kernel/linux/linux-yocto_5.15.25-rt33.bb

```diff
@@ -48,6 +48,7 @@ SRC_URI_append = " file://0041-enable-EXTCON_INTEL_MRFLD.cfg"
 
 # our additional configs
 SRC_URI_append = " file://ftdi_sio.cfg"
+SRC_URI_append = " file://ch341.cfg"
```

Comment:
I'd like to add "additional patches flying around to enable GPIO and I¹C on CH341 modules" but can't test myself. I don't mind to add if you tested or if at least they don't break the build. Which ones?

Reply:
I have no hardware to test, it was just a spontaneous idea. https://lore.kernel.org/all/20220401023306.79532-2-frank@zago.net/T/ Not sure if it’s the latest version of the series.

Reply:
I see this is brand new v5 = 1-apr-2022. I could add to Honnister or it will go into a new kernel anyway and maybe Marco V (on Telegram) can test?